### PR TITLE
tweak for django-2 support

### PR DESCRIPTION
**Pull request by sonlinux**
- [x] foreignkey attributes demands on delete property
- [x] reverse_lazy moved to urls package

### CODE REVIEW

**Comment by jmaupetit** [comment]
Hi @sonlinux! Thank you for this work :heart:

The CI seems broken, but it's not on you. I need to fix it first.

File: td_biblio/views.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 from django.contrib import messages
 from django.contrib.auth.decorators import login_required, user_passes_test
-from django.core.urlresolvers import reverse_lazy
+from django.urls import reverse_lazy
```
Comment (jmaupetit) [inline td_biblio/views.py]: Not sure this is compatible with older Django releases.

Comment (sonlinux) [reply td_biblio/views.py]: Ah yes let me have a check for this

File: td_biblio/views.py [hunk 2]
```diff
@@ -4,7 +4,10 @@
 
 from django.contrib import messages
 from django.contrib.auth.decorators import login_required, user_passes_test
-from django.urls import reverse_lazy
+try:
+    from django.core.urlresolvers import reverse_lazy
+except:
+    from django.urls import reverse_lazy
```
Comment (jmaupetit) [inline td_biblio/views.py]: :+1:

**Comment by jmaupetit** [comment]
Hey @sonlinux 👋  I needed to fix Django 2 compatibility in this project. This has been fully implemented in #31 (and the CI is now fixed). I'll release it soon! Thank you again for your contributions! 🙏

**Pull request closed without merging**